docs/operations_guide/Lite.md

```diff
@@ -2,81 +2,110 @@
 
 :::
 
 ## Nussknacker and K8s cluster
 
 ### Runtime container
-Nussknacker scenario is deployed as k8s [deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/). 
-By default, it creates [nussknacker-lite-runtime-app](https://hub.docker.com/r/touk/nussknacker-lite-runtime-app) runtime container with scenario json representation passed as config map.
+
+Nussknacker scenario is deployed as
+k8s [deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/).
+By default, it creates [nussknacker-lite-runtime-app](https://hub.docker.com/r/touk/nussknacker-lite-runtime-app)
+runtime container with scenario json representation passed as config map.
 Runtime container executes infinite loop responsible for polling events from kafka topic.
 
 ### Startup probes
+
 Nussknacker configures HTTP readiness and liveness healtcheck probes for runtime container.
 
 Readiness probe is used at first and has and impact for container being treated as ready.
 It's configuration should include e.g how many partition kafka topic has, what kind of metric provider is used
 By default, is checks if container is ready every second for 60 times.
 
-
 | Probe          | Type             | Default value |
 |----------------|------------------|---------------|
 | readinessProbe | periodSeconds    | 1s            |
 | readinessProbe | failureThreshold | 60 times      |
 | livenessProbe  | periodSeconds    | 3s            |
 | livenessProbe  | failureThreshold | 10 times      |
 
-
 ### Parallelism
-Tuning parallelism one can impact how many independent consumers are reading from kafka topic. 
 
-It's worth noticing that in general number of replicas count should not exceed number of kafka partitions because in such a case exceeding replica will stay idle due to kafka assignment strategies.
+Tuning parallelism one can impact how many independent consumers are reading from kafka topic.
+
+It's worth noticing that in general number of replicas count should not exceed number of kafka partitions because in
+such a case exceeding replica will stay idle due to kafka assignment strategies.
 
 ### Deployment strategy
 
-Nu uses `Recreate` deployment strategy together with `READ_COMMITED` `isolation.level` property to ensure transactional manner of processing events and to avoid
+#### Streaming
+
+Nu uses `Recreate` deployment strategy together with `READ_COMMITED` `isolation.level` property to ensure transactional
+manner of processing events and to avoid
 situation in which different deployments both reads events from the same topic.
 
+#### Request-Response
+
+For this processing mode we use `RollingUpdate` deployment strategy to ensure HA.
+
 ### Failure recovery
 
-In case of an unexpected errors e.g. network issue between kafka cluster and runtime nussknacker will try to reconnect after `waitAfterFailureDelay`
+In case of an unexpected errors e.g. network issue between kafka cluster and runtime nussknacker will try to reconnect
```

Comment:
why lower case for Kafka and Nussknacker? There are many occurances of "kafka" around ...